pkg/kubelet/server/server.go

```diff
@@ -378,14 +376,6 @@ func (s *Server) InstallDefaultHandlers(enableCAdvisorJSONEndpoints bool) {
 		compbasemetrics.HandlerFor(r, compbasemetrics.HandlerOpts{ErrorHandling: compbasemetrics.ContinueOnError}),
 	)
 
-	// deprecated endpoint which will be removed in release 1.20.0+.
-	s.addMetricsBucketMatcher("metrics/resource/v1alpha1")
-	v1alpha1ResourceRegistry := compbasemetrics.NewKubeRegistry()
-	v1alpha1ResourceRegistry.CustomMustRegister(stats.NewPrometheusResourceMetricCollector(s.resourceAnalyzer, stats.Config()))
-	s.restfulCont.Handle(path.Join(resourceMetricsPath, v1alpha1.Version),
```

Comment:
as you are removing this line, the entire module "k8s.io/kubernetes/pkg/kubelet/apis/resourcemetrics/v1alpha1" is not needed any longer. Consider removing

Reply:
Just looked over, it has this comment =): 

```
// TODO(RainbowMango): We don't need to maintain this package anymore.
//  This package will be remove in release 1.20.0+. More details please refer to https://github.com/kubernetes/kubernetes/pull/86282.
```

Reply:
Done. The whole package has been removed now.